### PR TITLE
Active Storage Preview Warnings

### PR DESCRIPTION
**Pull request by schneems**
- When using active storage set to `:local` emit a warning letting people know that the storage might not work as they expect.

- When no `ffmpeg` is available warn that they need to add a special buildpack.

### CODE REVIEW

File: lib/language_pack/rails5.rb [hunk 1]
```diff
@@ -30,4 +30,55 @@ def default_config_vars
...
+      out = run!('bin/rails runner "puts %Q{heroku_detecting_active_storage_config=#{Rails.application.config.active_storage.service}}"')
+      out =~ /heroku_detecting_active_storage_config=local/
+    end
+
+    def warn_local_storage
+      mcount("warn.activestorage.local_storage")
+      warn(<<-WARNING)
+You set your `config.active_storage.service` to :local in production.
+If you are uploading files to this app, they will not persist after the app
+is restarted. Heroku applications have an ephemeral files system. To
```
Comment (calebthompson) [inline lib/language_pack/rails5.rb]: after the app is restarted, on other web dynos, or in one off dynos.

People don't get this, and I guarantee we'll see tickets about it.

Comment (schneems) [reply lib/language_pack/rails5.rb]: Updated. Also I added more words in the linked article.

**Comment by calebthompson** [comment]
Thanks!

**Comment by dinatih** [comment]
Same here : 
```
       Running: rake assets:clean
       [2018-04-23T18:31:21 (process) #416][INFO] appsignal: Loading Rails (5.2.0.rc2) integration
       [2018-04-23T18:31:24 (process) #416][INFO] appsignal: Starting AppSignal 2.5.1 (vendor/bundle/bin/rake, Ruby 2.4.2, x86_64-linux)
       [2018-04-23T18:31:24 (process) #416][INFO] appsignal: Installing action_cable hook
       [2018-04-23T18:31:24 (process) #416][INFO] appsignal: Installing active_support_notifications hook
       [2018-04-23T18:31:24 (process) #416][INFO] appsignal: Installing net_http hook
       [2018-04-23T18:31:24 (process) #416][INFO] appsignal: Installing rake hook
sh: 2: Syntax error: Unterminated quoted string
 !
 !     Command: 'bin/rails runner "puts %Q{heroku_detecting_active_storage_config=#{Rails.application.config.active_storage.service}}"' failed unexpectedly:
 !     /tmp/build_378cbdf2243ef4ca11c806d5f6b4b029/config/environments/production.rb:52:in `+': no implicit conversion of nil into String (TypeError)
 !     from /tmp/build_378cbdf2243ef4ca11c806d5f6b4b029/config/environments/production.rb:52:in `block in <top (required)>'
 !     from /tmp/build_378cbdf2243ef4ca11c806d5f6b4b029/vendor/bundle/ruby/2.4.0/gems/railties-5.2.0.rc2/lib/rails/railtie.rb:216:in `instance_eval'
 !     from /tmp/build_378cbdf2243ef4ca11c806d5f6b4b029/vendor/bundle/ruby/2.4.0/gems/railties-5.2.0.rc2/lib/rails/railtie.rb:216:in `configure'
 !     from /tmp/build_378cbdf2243ef4ca11c806d5f6b4b029/config/environments/production.rb:1:in `<top (required)>'
```

**Comment by JPrevost** [comment]
I just opened https://github.com/heroku/heroku-buildpack-ruby/issues/744

**Pull request merged**